example.yml

```diff
@@ -129,6 +129,15 @@ modules:
     dns:
       query_name: "prometheus.io"
       query_type: "SOA"
+  dns_bind_version:
```

Comment:
This isn't really testing anything about the version. This is a niche enough use case that we can probably do without an example in the config.

Reply:
Removed :)